src/components/Kv/KvHero.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+	<div class="hero">
+		<div class="images">
+			<div class="images-placeholder"></div>
+			<div class="images-container">
+				<slot name="images"></slot>
+			</div>
+		</div>
+		<div class="headline" v-if="showHeadline">
+			<div class="headline-mask">
+				&nbsp;
+			</div>
+			<div class="headline-main">
+				<div class="headline-background" :style="headlineBgStyles"></div>
+				<div class="headline-title" v-if="hasHeadlineTitle">
+					<slot name="headlineTitle"></slot>
+				</div>
+				<div class="headline-body" v-if="hasHeadlineBody">
+					<slot name="headlineBody"></slot>
+				</div>
+				<div class="action" v-if="hasAction">
+					<slot name="action"></slot>
+				</div>
+			</div>
+			<div class="headline-mask">
+				&nbsp;
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		headlineBgColor: {
+			type: String,
+			default: ''
+		}
+	},
+	computed: {
+		hasHeadlineTitle() {
+			return !!this.$slots.headlineTitle;
+		},
+		hasHeadlineBody() {
+			return !!this.$slots.headlineBody;
+		},
+		hasAction() {
+			return !!this.$slots.action;
+		},
+		showHeadline() {
+			return this.hasHeadlineTitle || this.hasHeadlineBody || this.hasAction;
+		},
+		headlineBgStyles() {
+			return this.headlineBgColor ? `background-color: ${this.headlineBgColor}` : '';
+		}
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.hero {
+	position: relative;
+	width: 100%;
+	margin-bottom: 5rem;
+
+	@include breakpoint(medium) {
+		margin-bottom: 6rem;
+	}
+}
+
+.images {
+	position: relative;
+	overflow: hidden;
+
+	img {
+		width: 100%;
+	}
+}
+
+.images-container {
+	position: absolute;
+	top: 0;
+}
+
+.images-placeholder {
+	background-color: $kiva-bg-lightgray;
+	width: 100%;
+	padding-bottom: 600/480 * 100%;
```

Comment:
@ryan-ludwig Just curious, what's the value/purpose of setting this padding-bottom to a calculation instead of the hardcoded value of the calculation?

Reply:
I grabbed these styles from the homepage slider but I like how it's set up. It makes it a little more clear that this padding is determined by the width/height of the image inside (600px x 480px here), rather than seeing something arbitrary like 125%.

Reply:
@ryan-ludwig I just spoke to matt and he informed me that you were basing this off of the HeroSlideshow.vue file, where I also see these styles. 👍 Would be good to have a conversation with you around the definitions we have defined in _settings.scss and how to fold them into these reusable components. Having that discussion will also bring up some larger questions around, how do we fix our "styleguide" to be more useful and reuseable.

Reply:
Totally! I'd love to chat about common variables and future styleguide plans.